hack-the-resume/src/styles/App.css

```diff
@@ -0,0 +1,28 @@
+.App {
+  text-align: center;
+  display: flex;
+}
+
+.App-logo {
+  height: 40vmin;
+  pointer-events: none;
+}
+
+@media (prefers-reduced-motion: no-preference) {
+  .App-logo {
+    animation: App-logo-spin infinite 20s linear;
+  }
+}
+
+.App-link {
+  color: #61dafb;
+}
+
+@keyframes App-logo-spin {
```

Comment:
You might want to consider taking out this stuff - I believe this is from the react spinning logo when you first do npx create react app, might be confusing if someone is trying to read through this code!